project_task_send_by_mail/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Task Send By Mail",
+    "summary": "Send task report by email",
+    "version": "12.0.1.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "category": "Project Management",
+    "license": "AGPL-3",
+    "depends": [
+        "project_task_report",
```

Comment:
Should depends on project

Reply:
@hveficent No, we have used [report_template](https://github.com/OCA/project/pull/467/files#diff-5bc23089394fb44b1a07f1985af11e0dR11).

Here [project_task_report](https://github.com/OCA/project-reporting/tree/11.0/project_task_report)